### ISSUE TITLE
Source code?

### ISSUE THREAD

**Issue by nightmareci** [wipeout-phantom-edition/wipeout-phantom-edition#11]
Just wondering if you'll publicly put up the source code of this. Not demanding it, it's your choice to do so.

Making the source code public could lead to ports to platforms you're unable to support yourself, done by other developers with the necessary hardware/know-how.

**Comment by YesterPlay80** [comment]
You already can find the source code in the [releases](https://github.com/wipeout-phantom-edition/wipeout-phantom-edition/releases/latest).

**Comment by zaphod77** [comment]
have you actually downloaded that? :)

**Comment by nightmareci** [comment]
> You already can find the source code in the [releases](https://github.com/wipeout-phantom-edition/wipeout-phantom-edition/releases/latest).

The "Source code" downloads just contain the readme and images, no code. Those aren't uploaded by the repository owner, they're automatically generated by GitHub when you make a release. This repository only contains images and the readme, so the "Source code" download ends up only containing those.

**Comment by YesterPlay80** [comment]
> have you actually downloaded that? :)

Actually not, I just assumed that the archive named "source code" would actually contain the source code. :)

**Comment by vanfanel** [comment]
HUh, so.. no source code available then?

Too bad, was about to try building this on the Raspberry Pi. Seems to use SDL2 and OpenAL, so it will run EVERYWHERE if the sources are released.

**Comment by zaphod77** [comment]
There's almost certainly a reason the source is not available, but i won't
say more on the subject.

**Comment by sheepo99** [comment]
Kindly release the source, I really want to try compiling this on Linux. The code for the original was publicly leaked and Sony didn't care much for it, so there's very little danger here.

**Comment by roostercactus** [comment]
source (psx/windows anyway) was released by FoI and is on the archive if you want that.

**Comment by zaphod77** [comment]
Pretty sure Psygnosis owns the code, not Sony.

I expect that it will not get released untir permission is obtained, if
ever.

**Comment by YesterPlay80** [comment]
Psygnosis belonged to Sony since the early 1990s and was even renamed to Sony Computer Entertainment Studio Liverpool at a later point. Even later the studio was closed, though, so there is no more Psygnosis to give permission. The trademark and the code are 100% property of Sony, that's why after Wipeout 2097/XL there was never a Wipeout game on any non Sony system.

**Comment by vanfanel** [comment]
I have an idea. If this is based off the source code from 2022, which is broadly available, why not release the sources as a patch on that?
Using diff to recreate the complete souces locally and build for whatever we want would be trivial.

**Comment by zaphod77** [comment]
that should work.  interesting. sony still renewed their trademarks,
though.  so there oculd be future action.  While there are no developers
anymore, they still exist as a legal entity.

**Comment by vanfanel** [comment]
No future actions are possible against a diff patch. It's perfectly safe.

**Comment by YesterPlay80** [comment]
I'm no expert aon the matter and even though I'd like to see a proper Steam Deck release, I'd still like to point out that a diff file still contains part of the original code, so you still might get in trouble releasing one.

**Comment by eobet** [comment]
I wasn't even aware you could release something on Github without releasing the source code...

**Comment by roostercactus** [comment]
another port...but they have source (actual source!).  linux support as well.  https://github.com/phoboslab/wipeout-rewrite

**Comment by vanfanel** [comment]
@roostercactus yep, moved to that one. Works incredibly well on the Pi4 on Wayland and KMS/DRM, even with the CRT shader!

**Comment by JonSGoopple** [comment]
Would like to compile the source myself to ensure there are no trojan in the code as the .exe in the archive is detected by Windows Defender as ZkarletFlash trojan.

**Comment by coreybruce** [comment]
Yeah with the whole xz de buckle it makes this project look shady when there is only a compiled binary and no source code. I don't see a issue releasing the source code for such a old dead abandoned game from 1995
